tests/cross_functional/system_test/multicluster/test_post_installation_state.py

```diff
@@ -123,10 +123,13 @@ def test_ceph_clients(self):
                     ]
                     == consumer_name
                 ):
-                    found_client = (
-                        f"{cephclient['metadata']['annotations']['ocs.openshift.io.storageclaim']}-"
-                        f"{cephclient['metadata']['annotations']['ocs.openshift.io.cephusertype']}"
-                    )
+                    try:
+                        found_client = (
+                            f"{cephclient['metadata']['annotations']['ocs.openshift.io.storageclaim']}-"
+                            f"{cephclient['metadata']['annotations']['ocs.openshift.io.cephusertype']}"
+                        )
+                    except KeyError as err:
```

Comment:
This test is actually not needed anymore, so I reverted the changes. It will be deprecated after https://github.com/red-hat-storage/ocs-ci/pull/11027 is merged